index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Daily web components demo</title>
+    <link rel="icon" type="image/png" href="assets/favicon.png" />
+    <link rel="stylesheet" href="css/main.css" />
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <header>
+        <div class="header-section">
+          <img src="/assets/logo.svg" alt="Daily logo" />
+          <span class="title">Web Components demo</span>
+        </div>
+        <div class="header-section">
+          <a
+            class="new-tab-link"
+            href="https://docs.daily.co/reference/daily-js"
+            target="_blank"
+            rel="noreferrer noopenner"
+          >
+            <span>API docs</span>
+            <img src="/assets/newtab.svg" alt="New tab" />
+          </a>
+          <a
+            class="github-link"
+            href="https://github.com/daily-demos/svelte-prebuilt"
```

Comment:
Only other thing I noticed 👀 just needs a url update

Reply:
Haha woops, I really was like "let's make it EXACTLY like the svelte demo"